x-pack/plugins/rule_registry/server/utils/create_persistence_rule_type_wrapper.ts

```diff
@@ -49,6 +50,9 @@ const augmentAlerts = <T>({
         [ALERT_START]: currentTimeOverride ?? new Date(),
         [ALERT_LAST_DETECTED]: currentTimeOverride ?? new Date(),
         [VERSION]: kibanaVersion,
+        ...(options?.maintenanceWindowIds?.length
```

Comment:
Since these are persisted alerts, we do not need to update previous alert instances like we need to for AAD. We just need to make sure they reflect the current active maintenance windows